instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/BedrockRuntimeAccess.java

```diff
@@ -84,4 +91,25 @@ static Long getUsageInputTokens(SdkResponse response) {
   static Long getUsageOutputTokens(SdkResponse response) {
     return enabled ? BedrockRuntimeImpl.getUsageOutputTokens(response) : null;
   }
+
+  @NoMuzzle
+  static void recordRequestEvents(
+      Context otelContext, Logger eventLogger, SdkRequest request, boolean captureMessageContent) {
+    if (enabled) {
+      BedrockRuntimeImpl.recordRequestEvents(
+          otelContext, eventLogger, request, captureMessageContent);
+    }
+  }
+
+  @NoMuzzle
+  static void recordResponseEvents(
+      Context otelContext,
+      Logger eventLogger,
+      SdkResponse response,
+      boolean captureMessageContent) {
+    if (enabled) {
```

Comment:
I know this isn't new to this change, but curious if enabled is dynamic or not. If not then you could maybe guess where my head is at.

Reply:
It's a static final so should be fully optimized away, it's a pattern used in some other `Impl` classes too though yeah I may have gone with class selection instead.